packages/shared/src/styles/components/buttons.css

```
@@ -252,6 +255,7 @@
 
   &:hover, &.hover, &:focus:focus-visible {
     --button-icon: var(--button-hover-icon);
+    transition: all 0.2s ease-in-out;
```

Same goes here, just tagging for visibility 🚢 
```suggestion
    transition: background-color 0.2s ease-in-out;
```